InterventionSkillExample/main.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+from fastapi import FastAPI
+
+app = FastAPI()
+
+model = None
+
+@app.post('/invoke')
+def run(params: dict):
+
+    try:
+        payload = params["payload"]
+        profile = payload["profiles"]
```

Comment:
Is this a single profile or an array of profiles?

Reply:
At first we thought of sending an array of profileIDs. But, just in case if we pass a huge payload with a lot of profileIDs at once, cortex will not be able to handle it. We are going to use a single profile ID. Will rename it accordingly here.